lib/govuk_index/elasticsearch_presenter.rb

```diff
@@ -16,28 +16,42 @@ def identifier
     end
 
     def document
+      {}.merge(common_fields)
+        .merge(fields_from_expanded_links)
```

Comment:
We are leaving this for now as we feel that this should be considered a little later on perhaps when moving a format with parts or moving licenses